azure-template/tests/test_simple.py

```diff
@@ -0,0 +1,11 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
+from devtools_testutils import AzureMgmtTestCase
```

Comment:
This is a tough one, deserve discussion or at least link to
https://github.com/Azure/azure-sdk-for-python/wiki/Contributing-to-the-tests

Reply:
Done.